modules/wif-overview.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/creating-a-gcp-cluster-with-workload-identity-federation.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="workload-identity-federation-overview_{context}"]
+= Workload Identity Federation Overview
+
+Workload Identity Federation (WIF) allows {product-title} on {GCP} customers to use federated identities instead of service account keys when creating the necessary credentials needed to access GCP resources. Credentials created using WIF are considered short-term and more secure, while credentials created using service account keys are considered long-term and less secure. WIF minimizes the rights granted to third parties such as Red Hat, and ensures all Google Cloud native IAM and best practices for securing Google Cloud services and infrastructure are followed.
```

Comment:
Good question. This is a recommendation by Google...they suggest, for security reasons, to use WIF instead of SA authentication type. From Google docs: 
![image](https://github.com/user-attachments/assets/058f6276-eee3-41c0-a08a-3e130163314e)
That being said, can probably remove the word "considered"...